.circleci/config.yml

```diff
@@ -58,6 +58,7 @@ jobs:
     steps:
       - checkout
       - *restore_venv_cache
+      - *build
```

Comment:
updating the cache keys revealed that this was missing, but `restore_venv_cache` was providing the required dependencies